pkg/conversion/mapi2capi/aws.go

```diff
@@ -494,7 +490,7 @@ func blockDeviceMappingSpecToVolume(fldPath *field.Path, bdm mapiv1.BlockDeviceM
 
 	return capav1.Volume{
 		DeviceName:    ptr.Deref(bdm.DeviceName, ""),
-		Size:          *bdm.EBS.VolumeSize,
+		Size:          ebsVolumeSize,
```

Comment:
You could `ptr.Deref` here, and add a comment inline
```suggestion
		Size:          ptr.Deref(bdm.EBS.VolumeSize, 120), // The installer uses 120GiB by default as of 4.19.
```

Reply:
When you say invalidate? Do you mean, make it redundant? I guess that's fine no? We can delete the code?

AFAICT this would achieve the same thing that function is doing

Reply:
Ah, yes I was mistaken - it does do what the function does already. Thanks!